### PR TITLE
Blacklist for splitting multigame keys

### LINKED ISSUE #100: "Gremlins, Inc." parsed as two games "Gremlins (Steam)" and "Inc. (Steam)"
**Describe the bug**
The game "Gremlins, Inc." appears like this:
obrazek

It used to appear correctly some time ago. No other games have " (Steam)" appended to the name.

**Expected behavior**
It's a single game, "Gremlins, Inc.": 
obrazek

Probably an issue with handling the comma?

**Plugin installed**
directly from Galaxy

### PR DESCRIPTION
**Pull request by UncleGoogle**
Fixes #100 

tldr
Previously all keys that have coma in `human_name`s where splatted into multiple keys as this was the only way to extract all real games from one key using humble API.

This PR tries to avoid false positives by ignoring non-bundles and using blacklist for games that have "real" coma in their titles. In the future blacklist may be extended. Whitelist also maybe added in case there are keys that have multiple coma-separated games, but at least one of them is blacklisted.

### CODE REVIEW

File: src/consts.py [hunk 1]
```diff
@@ -28,3 +28,45 @@ class BITNESS(enum.Enum):
...
+    "Slow Down, Bull",
+    "Sorry, James",
+    "STAR WARS™ Battlefront (Classic, 2004)",
+    "STAR WARS™ Battlefront™ II (Classic, 2005)",
+    "That Dragon, Cancer",
+    "The Haunted Island, a Frog Detective Game",
+    "Total War Saga: THRONES OF BRITANNIA - Blood, Sweat and Spears",
+    "Upwards, Lonely Robot",
+    "Warhammer 40,000",
+    "Warhammer® 40,000",
```
Comment (JanPokorny) [inline src/consts.py:70]: You split by `", "` so you can drop these

File: src/consts.py [hunk 2]
```diff
@@ -28,3 +28,45 @@ class BITNESS(enum.Enum):
     CURRENT_BITNESS = BITNESS.B64
 else:
     CURRENT_BITNESS = BITNESS.B32
+
+COMA_SPLIT_BLACKLIST = [
```
Comment (JanPokorny) [inline src/consts.py:32]: hate to be pedantic but it's spelled "COMMA" 😅

File: src/consts.py (hunk 1 above)
Comment (UncleGoogle) [reply src/consts.py]: Right

File: src/consts.py (hunk 2 above)
Comment (UncleGoogle) [reply src/consts.py]: No worries, it is important for reader aesthetic feelings ;)

**Pull request merged**